### PR TITLE
Add Android documentation

### PR DESCRIPTION
**Pull request by lol768**

### CODE REVIEW

File: docs/android.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+package com.example.kiclandoridtest;
+
+import android.os.AsyncTask;
+
+import org.kitteh.irc.client.library.Client;
+
+public class ConnectIrcTask extends AsyncTask<Void, Void, Void> {
+    @Override
+    protected Void doInBackground(Void... voids) {
+        Client client = Client.builder().nick("KittehAndroid").server().host("irc.esper.net").then().buildAndConnect();
```
Comment (Zarthus) [inline docs/android.md]: please use a fake network in documentation that does not resolve

File: docs/android.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+### Ensure minSdkVersion >= 24
+
+In `app/build.gradle`:
+
+```groovy
+android {
+    // ...
+    defaultConfig {
+        applicationId "com.example.kiclandoridtest"
```
[the remark this answers is not in the document]
Comment (Zarthus) [reply docs/android.md]: rookie mistake

File: docs/android.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    protected Void doInBackground(Void... voids) {
+        Client client = Client.builder().nick("KittehAndroid").server().host("localhost").then().buildAndConnect();
+
+        client.addChannel("#kittehandroid");
+        client.sendMessage("#kittehandroid", "Hello World!");
+        return null;
+    }
+}
+```
+
```
Comment (Zarthus) [inline docs/android.md]: two newlines!!!!

**Comment by mbax** [comment]
Thanks! https://github.com/KittehOrg/KittehIRCClientLib/commit/be5ec31d009ca05ddd15280e25e27967853f12b9

**Pull request closed without merging**